Makefile

```diff
@@ -19,6 +19,8 @@ GO = go
 GOBIN = $(PWD)/bin
 export GOBIN
 
+vendor-id=
```

Comment:
Since we are also using other variables in this file (see above this line), I think it's better to do something like this:

```
VENDOR_ID=
```

and then document as:

```bash
VENDOR_ID=myvendor make validate
```